plugins/lookup/github_app_access_token.py

```diff
@@ -154,22 +161,24 @@ def encode_jwt(app_id, private_key_obj, exp=600):
         raise AnsibleError(f"Error while encoding jwt: {e}")
 
 
-def post_request(generated_jwt, installation_id):
-    github_api_url = f'https://api.github.com/app/installations/{installation_id}/access_tokens'
+def post_request(generated_jwt, installation_id, api_base=None):
```

Comment:
In the context of this function, `api_base` is mandatory.
```suggestion
def post_request(generated_jwt, installation_id, api_base):
```

Reply:
change applied in latest commit